rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -562,6 +566,7 @@ def list(self,
         :param get_all_results: Get all the results without the limitation of
                                 size or pagination. Use it carefully to
                                 prevent consumption of too much memory
+        :param distinct: If True, distinct ressults will return
```

Comment:
I will be taking that s back, for better results.

Reply:
Thanks! Changing